lib/ip_record.js

```diff
@@ -150,6 +151,14 @@ module.exports = function (limits, now) {
     this.sms = this.sms.slice(i + 1)
   }
 
+  IpRecord.prototype.addAccountAccess = function (now) {
+    this.aa.push(now)
+  }
+
+  IpRecord.prototype.isOverAccountAccessLimit = function (now) {
+    return this.aa.length > limits.maxAccountAccess
```

Comment:
Other `isOver...` methods trim their array but I don't see any need to do that here because the array gets truncated in `rateLimit` anyway.